example_crud.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+
+import etesync as api
+
+
+class EtesyncCRUD:
+    def __init__(self, email, userPassword, remoteUrl, uid, authToken,
+                 cipher_key=None):
+        r"""Initialize EtesyncInterface
+
+        Parameters
+        ----------
+        email : etesync username(email)
+        userPassword : etesync encryption password
+        remoteUrl : url of etesync server
+        uid : uid of calendar
+        authToken : authentication token for etesync server
+        """
+        self.etesync = api.EteSync(email, authToken, remote=remoteUrl)
+        if cipher_key:
+            self.etesync.cipher_key = cipher_key
+        else:
+            self.etesync.derive_key(userPassword)
+        self.journal = self.etesync.get(uid)
+        self.calendar = self.journal.collection
+        self.error = None
+
+    def create_event(self, event):
+        r"""Create event
+
+        Parameters
+        ----------
+        event : iCalendar file as a string
+        (calendar containing one event to be added)
+
+        Returns
+        -------
+        True if event created
+        False otherwise (Error is stored in self.error)
+        """
+        try:
+            ev = api.Event.create(self.journal.collection, event)
+            ev.save()
+            return True
+        except Exception as inst:
```

Comment:
Why do this and not just let the exception rise? I think it's cleaner to just have the exception there. That's what exceptions are for, exceptions...

Reply:
OK. This was a left over of the days when this ran as a daemon and daemons cannot raise.